### ISSUE TITLE
[$20][MSFT] Challenges page: The topcoder logo doesn't have an alt text.

### ISSUE THREAD

**Issue by Oanh-and-only-Oanh** [topcoder-platform/community-app#3188]
Steps to Reproduce

Go to the Challenges page: https://www.topcoder.com/challenges. 2. Ensure screen reader is switched on. 3. Press the shortcut 'ctrl + home' to make sure that you are at the top of the page. 4. Press the down arrow. 5. Notice screen reader reads 'link to Topcoder Homepage'
Expected Result
Screen reader should announce the logo as 'Graphic topcoder logo link to homepage'

Actual Result
Screen reader announces the topcoder logo incorrectly.

Device: Desktop/Labtop
Operating System: Windows
Browser: Chrome

WCAG Category: 1. Perceivable
WCAG Level: A
WCAG Success Criterion: 1.1.1 Non-Text Content
Screen Reader Used: JAWS
Accessibility Audit Tool used: N/A

Attachment
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2029

**Comment by Oanh-and-only-Oanh** [comment]
Reference: https://github.com/topcoder-platform/community-app/issues/2817

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30098337 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30098337 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
@rashmi73  Have you worked on this ticket already?  If not, please let us know, we accidentally made a duplicate ticket of this and someone else is already working on it.  If you already done work and it successfully solves the issue, we will still pay you for your efforts.  If not, we will just close this ticket.  Thank you!

**Comment by rashmi73** [comment]
@crazyk07 yes actualy it was completed, and was going to raise PR but due to power cut in my city due to heavy rain, I could not. Shall i still raise PR?as I see PR will be same as 8217 with just minor change of text.

**Comment by rashmi73** [comment]
@veshu PR ready https://github.com/topcoder-platform/community-app/pull/3192

**Comment by veshu** [comment]
@tosha5252 @drasticdpk Please confirm this is fixed by https://github.com/topcoder-platform/community-app/issues/2817

**Comment by tosha5252** [comment]
@veshu Yes..I have just tested that not long ago...it is fixed

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
